src/components/TodoItem.jsx

```diff
@@ -1,4 +1,17 @@
+import { useContext } from "react";
+import { TodoDeleteContext } from '../App';
+
 export const TodoItem = ({id, todoText, todoLimit, todoStatus}) => {
+
+  // タスク削除用の関数をuseContextで読み込む
+  const handleTodoDeleteFromContext = useContext(TodoDeleteContext);
+  
+  const deleteTodo = (deleteEvent) => {
+    const deleteId = {id};
+    deleteEvent.preventDefault();
+    handleTodoDeleteFromContext(deleteId);
```

Comment:
deleteIdをconstで定義し直す必要はないかと思いました

```suggestion
    deleteEvent.preventDefault();
    handleTodoDeleteFromContext(id);
```

Reply:
コミット「[75f83bc](https://github.com/hideki1516/react-todoapp/pull/3/commits/75f83bc438b45278fd036032d88227426012798a)」で修正しました。